Dockerfile

```diff
@@ -6,6 +6,8 @@
 
 # @not-generated
 
+# cSpell:ignore mikefarah
```

Comment:
Personally, I would rather consolidate these in our `.cspell.json`.

Reply:
ok if we keep `yq`, but it's not sure ;-)